CONTRIBUTING.md

```diff
@@ -29,19 +29,32 @@ If you see an issue that you'd like to fix feel free to do so. If possible let u
 
 Skosmos PHP code should follow [PSR-12](https://www.php-fig.org/psr/psr-12/) style. To help achieve this, [PHP-CS-Fixer](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer), a tool to automatically fix and verify PHP code style issues, is included as a development dependency that is installed via Composer.  There is also a code style check in the GitHub Actions CI pipeline that verifies the code style compliance.
 
-You can set up a pre-commit hook to automate style checking with PHP-CS-Fixer for every git commit by adding the following script to the file `.git/hooks/pre-commit`, which should have execute permission set:
+We expect good coding practices also for front-end code. Among many others, we use [Standard JS](https://standardjs.com/) to validate the JavaScript code.
+
+You must set up a pre-commit hook to automate style checking with PHP-CS-Fixer for every git commit by adding the following script to the file `.git/hooks/pre-commit`, which should have execute permission set. Pre-commit hook for Standard JS validation should also be added (more details about Standard JS in the text below).
 
 ```bash
 #!/bin/bash
 
 set -e
 vendor/bin/php-cs-fixer fix --diff --dry-run src
+
+JS_FILES_TO_BE_CHECKED="/installation_folder/skosmos/*.js /installation_folder/skosmos/some_other_folder/*.js"
+npx standard $JS_FILES_TO_BE_CHECKED
+
 ```
 
-If the hook complains and intercepts the commit, you can run PHP-CS-Fixer manually to reformat the PHP code using this command:
+If the hook complains about the PHP code styles and intercepts the commit, you can run PHP-CS-Fixer manually to reformat the PHP code using this command:
 
     vendor/bin/php-cs-fixer fix src
 
+Fixing the errors reported by Standard JS, we suggest that style errors be fixed using the --fix flag. For example:
+```standard --fix *.js``` (automatically makes changes to files).
+
+Standard JS is part of the Skosmos installation package, so you do not need to do any additional installation after ```npm install```.
+
+If PHP-CS-Fixer or Standard JS work unexpectedly, please let us know in the Skosmos [Users group](https://groups.google.com/g/skosmos-users).
```

Comment:
I think this is a bit redundant, because the user forum is already mentioned at the top of CONTRIBUTING.md, and maybe the users forum is not the best place to discuss details of Skosmos development. I would just drop this.

Reply:
Removed